doc/ref/tags/tag_image.rst

```diff
@@ -37,6 +37,8 @@ The following arguments/filters can be specified:
 |mediaclass          |The media class of the image. See                           |mediaclass="thumb"  |
 |                    |:ref:`manual-media-classes`.                                |                    |
 +--------------------+------------------------------------------------------------+--------------------+
+|srcset              |Provide the image in multiple `responsive sizes`_           |srcset=["150", "450"]|
```

Comment:
Is this list format a good way to provide the arg? Or is something like `srcset=150 srcset=450` preferred?